mypyc/ir/ops.py

```diff
@@ -232,17 +232,23 @@ def format(self, fmt: str, *args: Any) -> str:
                 i = n
         return ''.join(result)
 
-    def to_lines(self) -> List[str]:
+    def to_lines(self, const_regs: Dict[str, int] = {}) -> List[str]:
```

Comment:
Nit: It's considered error-prone to have a mutable default value. It's better to use `None` as the default and construct an empty dictionary if needed in the body. Some linters complain about this. (I know it's semantically fine here, but I think it's better to avoid this for style reasons.)